### PR TITLE
added APK in demo apk folder

### LINKED ISSUE #26: Prepare a debug build of the sample app and store it in a demo-apk folder in the root of the repo
Also, update the readme.md with a link directly to this apk

### PR DESCRIPTION
**Pull request by sumanthst24**
BarGroup(Context context, String labelText, String color, String valueText, float progress)  constructor in BarGroup.java is never used

### CODE REVIEW

**Review by sumanthst24 — COMMENTED** [review]
added demo apk file in demo apk folder

File: README.md [hunk 1]
```diff
@@ -88,3 +88,4 @@ barModels.add(new BarModel(
 
 barview.setData(barModels);
 ```
+The demo apk for the project is available demo apk folder of the root directory
```
Comment (krharsh17) [inline README.md]: Add a direct link to the apk here

File: demo apk/output.json [hunk 2]
```diff
@@ -0,0 +1 @@
+[{"outputType":{"type":"APK"},"apkData":{"type":"MAIN","splits":[],"versionCode":1,"versionName":"1.0","enabled":true,"outputFile":"app-debug.apk","fullName":"debug","baseName":"debug"},"path":"app-debug.apk","properties":{}}]
```
Comment (krharsh17) [inline demo apk/output.json]: Delete this file

File: library/src/main/java/in/krharsh17/barview/BarGroup.java [hunk 3]
```diff
@@ -31,19 +31,6 @@
...
-        this.labelText = labelText;
-        this.color = color;
-        this.valueText = valueText;
-        this.progress = progress;
-
-        label = new TextView(context);
-        initial = new View(context);
-        bar = new Bar(context);
-        value = new TextView(context);
-    }
```
Comment (krharsh17) [inline library/src/main/java/in/krharsh17/barview/BarGroup.java:46]: Why has this been done?

**Review by krharsh17 — CHANGES_REQUESTED** [review]
Also, the name of the folder has to be `demo-apk` instead of `demo apk`. Whitespaces should not be used in folder naming.

File: README.md [hunk 4]
```diff
@@ -88,3 +88,4 @@ barModels.add(new BarModel(
 
 barview.setData(barModels);
 ```
+The demo apk for the project is available at barview-android/demo-apk/
```
Comment (krharsh17) [inline README.md]:
```suggestion
The demo apk for the project is available [here](https://github.com/krharsh17/barview-android/blob/develop/demo-apk/app-debug.apk)
```

**Review by krharsh17 — CHANGES_REQUESTED** [review]

**Review by krharsh17 — APPROVED** [review]

**Pull request merged**